music_assistant/server/providers/deezer/__init__.py

```diff
@@ -615,7 +615,7 @@ def parse_track(
         if hasattr(track, "artist"):
             artist = ItemMapping(
                 media_type=MediaType.ARTIST,
-                item_id=str(track.artist.id),
+                item_id=str(getattr(track.artist, "id", track.artist.name)),
```

Comment:
```suggestion
                item_id=str(getattr(track.artist, "id", f"deezer-{track.artist.name.lower()}")),
```

maybe just this and call it a day ?

Reply:
Yeah looks good to me. The only thing I can think of is whether it should be `deezer-user` to make it clear that it's from a user-uploaded track, but I'll commit this and let you decide whether to merge or change.